.github/workflows/global_move_bot_pr_to_board.yml

```diff
@@ -0,0 +1,145 @@
+# This workflow runs on a schedule
+# It finds all the PR owned by neurobagel
+# that have the _bot label applied.
+# It then checks if the PR has already been added to 
+# the main project and adds the PR if it hasn't been added yet.
+# Finally the workflow moves the PR to the "Community" column
+# on the project board.
+
+name: Scheduled PR automation
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '30 5,17 * * *'
+  workflow_dispatch:
+
+jobs:
+  get_prs:
+    name: Get all PRs with the bot label
+    runs-on: ubuntu-latest
+    outputs:
+      urls: ${{ steps.get_pr.outputs.pr_urls }}
+    steps:
+      - name: Get current repositories
+        id: get_pr
+        env:
+          GH_TOKEN: ${{ secrets.LAB_PAT  }}
+          MAX_REPO: 50
+        run: |
+          pr_urls=$(gh search prs --owner neurobagel label:_bot --state open --json 'url' | jq -c '[.[].url]')
+          echo "pr_urls=${pr_urls}" >> $GITHUB_OUTPUT
+
+  process_prs:
+    name: Process PR
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.NB_PROJECT_PAT }}
```

Comment:
Interesting. Maybe this is what you were saying in the issue description, but does this mean that the PAT can't be the same for the `get_prs` job and the `process_prs` job?

Reply:
Yes, I'll add a comment